prow/config.gen.yaml

```diff
@@ -1175,7 +1175,29 @@ presubmits:
           requests:
             cpu: "4"
             memory: 4Gi
-
+  maistra/prometheus:
+  - name: prometheus-test
+    decorate: true
+    branches:
+      - maistra-2.1
+    max_concurrency: 2
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.1"
+        imagePullPolicy: Always
+        command:
+        - make
+        - build
+        - test
+        securityContext:
+          privileged: true
```

Comment:
sorry, just saw this. Does it need this privilege?

Reply:
I don't think so. Removed. Can always add it back if we need it.